drivers/i3c/i3c_ccc.c

```diff
@@ -591,3 +591,110 @@ int i3c_ccc_do_getcaps(const struct i3c_device_desc *target,
 out:
 	return ret;
 }
+
+int i3c_ccc_do_setvendor(struct i3c_device_desc *target, uint8_t id, uint8_t *payload, size_t len)
+{
+	struct i3c_ccc_payload ccc_payload;
+
+	__ASSERT_NO_MSG(target != NULL);
+
+	/* CCC must be between 0xE0 and 0xFE */
+	if (id > 0x1E) {
+		return -EINVAL;
+	}
```

Comment:
shouldn't this check for < 0xE0 and > 0xFE according to the comment?

Reply:
no, `id` used with the macro `I3C_CCC_VENDOR`
```c
#define I3C_CCC_VENDOR(broadcast, id)		((id) + ((broadcast) ? 0x61U : 0xE0U))
```